### PR TITLE
add the Spanish translation of using channels

### PR DESCRIPTION
**Pull request by come-maiz**
Notes for reviewers:

* [Video](https://www.youtube.com/watch?v=-3b9qkl9Z_k)
* [English description](https://github.com/elopio/snapcraft-videos/blob/master/using_channels/description.txt)
* [English subtitles](https://github.com/elopio/snapcraft-videos/blob/master/using_channels/subtitles.srt)
* Please leave your review comments in English.

### CODE REVIEW

File: using_channels/subtitles-es.srt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+00:00:40,960 --> 00:00:44,700
+Sus equipos de integración contínua y aseguramiento de la calidad pueden subscribirse a «edge»
+
+14
+00:00:44,700 --> 00:00:47,480
+para obtener las versiones más recientes de forma automática.
+
+15
+00:00:47,660 --> 00:00:49,920
+Durante la preparación de una versión puede usar el canal «beta»
```
Comment (allanesqui) [inline using_channels/subtitles-es.srt]: During prerelease you can use the beta channel > Durante el pre-lanzamiento usted puede usar el <canal beta>

File: using_channels/subtitles-es.srt [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+00:01:00,360 --> 00:01:02,040
+se moverán con usted.
+
+21
+00:01:02,120 --> 00:01:05,980
+De forma opcional, también puede usar el canal «candidate» antes de una publicación.
+
+22
+00:01:06,280 --> 00:01:09,860
+Los canales están diseñados para ser flexibles y soportar su proceso de desarrollo.
```
Comment (allanesqui) [inline using_channels/subtitles-es.srt]: support your release process > soportatar su proceso de lanzamiento o (publicación)

**Review by allanesqui — COMMENTED** [review]
Agrego sugerencias de traducción.

**Comment by come-maiz** [comment]
Thank you @allanesqui!
I've included your suggestions. Please approve the PR if it now looks good to you.

**Review by allanesqui — APPROVED** [review]
I have reviewed again. Seems good.

**Pull request merged**